release-notes/README.md

```diff
@@ -4,6 +4,35 @@ description: Release notes and change logs for the Adobe Experience Platform Mob
 
 # Release notes
 
+## Sept 3, 2021
+
+### iOS AEPCore 3.3.0
+
+* Enables dispatch rules for event data forwarding.
+
+### iOS AEPLifecycle 3.3.0
+
+* Lifecyle extension now supports Mobile Lifecycle metrics in XDM format that can be forwarded to the Edge Network when using Adobe Launch rules.
```

Comment:
Should we mention 2 new events? (applicationLaunch and applicationClose)

Reply:
Lifecycle extension now dispatches 2 new events applicationLaunch and applicationClose which contain Mobile Lifecycle metrics in XDM format.

Reply:
We can talk about forwarding the events in the page documenting Lifecycle Edge workflow.